### ISSUE TITLE
Select converts returned keys to string

### ISSUE THREAD

**Issue by jonathon-m** [tensor-labs/rocket-edit#3]
Using select options:
```
yesNoOptions = [{key: true, value: 'Yes'}, {key: false, value: 'No'}];
```
And template:
```
<rocket-edit type="select" (Save)="save()" [(ngModel)]="yesNoKey"
                selectPlaceholder="Not Set" [options]="yesNoOptions"></rocket-edit>
```

yesNoKey should be returned as `true` or `false`, but is instead returned as `"true"` or `"false"`.

I've had a look, but can't identify where this is occurring.

**Comment by yadavgvijay** [comment]
This issue is fixed in npm version 0.1.8. 
@jonathon-m Please verify and close the issue.

**Comment by jonathon-m** [comment]
Fixed! Thanks @yadavgvijay

**Comment by jonathon-m** [comment]
This issue has re-appeared in version 1.0.0

**Comment by yadavgvijay** [comment]
This issue is fixed in npm version 0.1.8.
@jonathon-m Please verify and close the issue.

**Comment by yadavgvijay** [comment]
@jonathon-m closing this issue as there is no response from you.
